ui/src/router/activity_log/events_list/EventsList.scss

```diff
@@ -0,0 +1,3 @@
+.activityLogEvent {
```

Comment:
This is so that the "\n" in the activity log event is rendered as a newline (every Type.. is a new line).
![Screenshot 2020-12-03 at 4 18 14 PM](https://user-images.githubusercontent.com/23465343/100982938-9fa04200-3583-11eb-9d81-07bd5a3681b9.png)